__tests__/bundle/info-description-override-error/.redocly.yaml

```diff
@@ -1,7 +1,8 @@
 apis:
   main:
     root: ./main.yaml
-
+rules:
```

Comment:
Aren't these tests for testing `bundle`? I'd avoid adding rules here as they seems redundant.

Reply:
Okay, i removed `--lint` option (where it is not needed) in bundle tests